python/pyspark/ml/regression.py

```diff
@@ -1258,9 +1262,14 @@ class GeneralizedLinearRegression(JavaEstimator, HasLabelCol, HasFeaturesCol, Ha
     ...     (1.0, Vectors.dense(1.0, 2.0)),
     ...     (2.0, Vectors.dense(0.0, 0.0)),
     ...     (2.0, Vectors.dense(1.0, 1.0)),], ["label", "features"])
-    >>> glr = GeneralizedLinearRegression(family="gaussian", link="identity")
+    >>> glr = GeneralizedLinearRegression(family="gaussian", link="identity", linkPredictionCol="p")
+    >>> print(glr.getLinkPredictionCol())
```

Comment:
minor, but is this really necessary? By definition it would be set correctly if the test below succeeds `>>> abs(transformed.head().p - 1.5) < 0.001`